### ISSUE TITLE
Using AWT Windows in Stand-Alone Applications 

### ISSUE THREAD

**Issue by benardali** [section-engineering-education/engineering-education#4519]
#### Pre-submission advice
By following all our pre-submission advice and reviewing our Resources folder, you will maximise your chances of your topic being approved.

We ask that you please be patient as our team works through approving and publishing all articles/tutorials in a timely manner.

Allow 1-3 days for a topic to be reviewed and/or approved - allow 3-7 days for an articles to be reviewed and/or published.

Be sure to visit our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for tools, resources, and example articles that will help you propose and write a successful article.

**Please ensure that you have only one open issue + linked pull request at a time. This will ensure that we complete the article in a timely manner from inception to publishing.)**

**We tend to stray away or tend not to publish reviews/comparisons of commercial product offerings.**

#### Proposed title of article
Using AWT Windows in Stand-Alone Applications 

#### Proposed article introduction
In this article I will discuss the AWT windows in java and the importance and how to use it in developing Stand-Alone Applications.
I will come up with some codes demonstrating the use of AWT windows in Stand-Alone Appication
#### Key takeaways 
By the end of this tutorial the learner will be able to:

- Understand the basis of AWT.
- Be able to write a code to develop Stand-Alone Application in Java.

#### Article quality
The quality of the article is moderate and good since some of the blog sites does not tackle such.

#### References
N/A

### Conclusion
In conclusion I hope the article will be helpful to the learners who have ideas in java programming.
### Templates to use as guides
- [How To Guide Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/how-to-guide-template.md)
- [Software Review Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/software-review-template.md)
- [Tutorial Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md)
